packages/tabs/src/elements/Tabs.tsx

```diff
@@ -36,9 +35,10 @@ export interface ITabsProps extends Partial<ThemeProps<DefaultTheme>> {
  */
 const Tabs = React.forwardRef<HTMLDivElement, ITabsProps>(
   (
-    { isVertical, children, onChange, selectedItem: controlledSelectedItem, theme, ...otherProps },
+    { isVertical, children, onChange, selectedItem: controlledSelectedItem, ...otherProps },
     ref
   ) => {
+    const theme = useContext(ThemeContext);
```

Comment:
I updated the theme reference logic here to align with the more recent components. This doesn't affect the public API.